client/src/js/analyses/__tests__/selectors.test.js

```diff
@@ -1,10 +1,19 @@
-import { getFilterIds, getMatches, getResults, getSortIds } from "../selectors";
+import { getMaxReadLength } from "../../samples/selectors";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getMaxReadLength' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5085484)